pkg/server/home_workspaces.go

```diff
@@ -137,7 +140,11 @@ func buildExternalClientsAccess(kubeClusterClient kubernetesclient.ClusterInterf
 			return err
 		},
 		createClusterWorkspace: func(ctx context.Context, workspace logicalcluster.Name, cw *tenancyv1alpha1.ClusterWorkspace) error {
-			_, err := kcpClusterClient.Cluster(workspace).TenancyV1alpha1().ClusterWorkspaces().Create(ctx, cw, metav1.CreateOptions{})
+			client := bootstrapKcpClusterClient
+			if cw.Spec.Type.Name == HomeClusterWorkspaceType {
+				client = kcpClusterClient
```

Comment:
why?

Reply:
i had issues with the owner annotation being set/not set. will go back and see if i can undo this, or add a comment for why